Samples/360VideoPlayback/cpp/Content/VideoRenderer.cpp

```diff
@@ -97,18 +97,17 @@ void VideoRenderer::Render()
         m_modelConstantBuffer.GetAddressOf()
     );
 
-    if (!m_usingVprtShaders)
-    {
-        // On devices that do not support the D3D11_FEATURE_D3D11_OPTIONS3::
-        // VPAndRTArrayIndexFromAnyShaderFeedingRasterizer optional feature,
-        // a pass-through geometry shader is used to set the render target
-        // array index.
-        context->GSSetShader(
-            m_geometryShader.Get(),
-            nullptr,
-            0
-        );
-    }
+    // On devices that do not support the D3D11_FEATURE_D3D11_OPTIONS3::
```

Comment:
It would be more clear, and potentially safer as well, to have an if-else case where the else case sets nullptr directly instead of relying on m_geometryShader.get() to be nullptr.

Reply:
Applied your suggestion.